src/demo-app/radio/radio-demo.ts

```diff
@@ -1,6 +1,7 @@
 import {Component} from '@angular/core';
+import {FORM_DIRECTIVES, NgFor} from '@angular/common';
```

Comment:
We should probably import FORM_DIRECTIVES from @angular/forms, no?  Unless you want that to be added to the open PR.